### PR TITLE
cached-tables: add description about `mysql.table_cache_meta` table

### PR DESCRIPTION
**Pull request by tiancaiamao**
### First-time contributors' checklist 

- [X] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Close https://github.com/pingcap/docs-cn/issues/10087

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [X] master (the latest development version)
- [X] v6.2 (TiDB 6.2 versions)
- [X] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): https://github.com/pingcap/docs-cn/issues/10087

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by tangenta** [comment]
We may need to update https://docs.pingcap.com/zh/tidb/dev/mysql-schema as well.

File: cached-tables.md [hunk 1]
```diff
@@ -177,6 +177,22 @@ SELECT * FROM users;
 >
 > 缓存表的写入延时高是受到实现的限制。存在多个 TiDB 实例时，一个 TiDB 实例并不知道其它的 TiDB 实例是否缓存了数据，如果该实例直接修改了表数据，而其它 TiDB 实例依然读取旧的缓存数据，就会读到错误的结果。为了保证数据正确性，缓存表的实现使用了一套基于 lease 的复杂机制：读操作在缓存数据同时，还会对于缓存设置一个有效期，也就是 lease。在 lease 过期之前，无法对数据执行修改操作。因为修改操作必须等待 lease 过期，所以会出现写入延迟。
 
+缓存表相关的元信息存储在 `mysql.table_cache_meta` 表中。这张表记录了所有缓存表的 `tid`，当前状态 `lock_type`，以及 `lease` 相关的信息。这张表仅供 TiDB 内部使用，用户修改可能导致不可预期的错误。
```
Comment (Oreoxmt) [inline cached-tables.md:180]:
```suggestion
缓存表相关的元信息存储在 `mysql.table_cache_meta` 表中。这张表记录了所有缓存表的 `tid`，当前状态 `lock_type`，以及 `lease` 相关的信息。

> **注意：**
>
> 这张表仅供 TiDB 内部使用，用户修改可能导致不可预期的错误。
```

Comment (tangenta) [inline cached-tables.md:180]:
```suggestion
缓存表相关的元信息存储在 `mysql.table_cache_meta` 表中。这张表记录了所有缓存表的 ID，当前的锁状态 `lock_type`，以及锁租约 `lease` 相关的信息。这张表仅供 TiDB 内部使用，用户修改可能导致不可预期的错误。
```

Comment (tiancaiamao) [reply cached-tables.md]: Why not tell this to the user?

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Review by tangenta — APPROVED** [review]

File: cached-tables.md [hunk 2]
```diff
@@ -177,6 +177,22 @@ SELECT * FROM users;
 >
 > 缓存表的写入延时高是受到实现的限制。存在多个 TiDB 实例时，一个 TiDB 实例并不知道其它的 TiDB 实例是否缓存了数据，如果该实例直接修改了表数据，而其它 TiDB 实例依然读取旧的缓存数据，就会读到错误的结果。为了保证数据正确性，缓存表的实现使用了一套基于 lease 的复杂机制：读操作在缓存数据同时，还会对于缓存设置一个有效期，也就是 lease。在 lease 过期之前，无法对数据执行修改操作。因为修改操作必须等待 lease 过期，所以会出现写入延迟。
 
+缓存表相关的元信息存储在 `mysql.table_cache_meta` 表中。这张表记录了所有缓存表的 ID，当前的锁状态 `lock_type`，以及锁租约 `lease` 相关的信息。这张表仅供 TiDB 内部使用，用户修改可能导致不可预期的错误。
```
Comment (TomShawn) [inline cached-tables.md:180]:
```suggestion
缓存表相关的元信息存储在 `mysql.table_cache_meta` 表中。这张表记录了所有缓存表的 ID、当前的锁状态 `lock_type`，以及锁租约 `lease` 相关的信息。这张表仅供 TiDB 内部使用，不建议用户修改该表，否则可能导致不可预期的错误。
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**